src/app/counter-analytics/counter-analytics.page.html

```diff
@@ -0,0 +1,22 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button icon="close" defaultHref="/counters"></ion-back-button>
+    </ion-buttons>
+    <ion-title>
+      <ion-icon name="analytics"></ion-icon>
+      {{ counter.title }}
+    </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+
+  <ion-list>
+    <ion-item>
+      <ion-label [routerLink]="['/counter-analytics', counter.id, 'day-of-week-histogram']">
```

Comment:
I would make this similar to what ionic has on its preview: https://ionicframework.com/docs/api/list , [[source]](https://github.com/ionic-team/ionic-docs/blob/master/src/demos/api/list/index.html)
So I would add an ion-avatar and an inside the ion-label `<h2>` for the title and `<p>` tags for a brief description.

Also I would keep the title and the description with the `DayOfWeekHistogramPage`. So maybe we could enforce via an interface or super class a `title` and description property for all those visualization pages?

Reply:
I like both suggestions.

Reply:
Done.